services/venice-controller/src/main/java/com/linkedin/venice/controller/ZkHelixAdminClient.java

```diff
@@ -164,7 +164,7 @@ public void addVeniceStorageClusterToControllerCluster(String clusterName) {
           AutoRebalanceStrategy.class.getName());
       VeniceControllerClusterConfig config = multiClusterConfigs.getControllerConfig(clusterName);
       IdealState idealState = helixAdmin.getResourceIdealState(controllerClusterName, clusterName);
-      idealState.setMinActiveReplicas(controllerClusterReplicaCount);
+      idealState.setMinActiveReplicas(controllerClusterReplicaCount - 1);
```

Comment:
Can we do `Math.max(controllerClusterReplicaCount - 1,1)` instead?

Reply:
Do we want this for the storage cluster too?

Reply:
We should have it there as well